django_project_491/django_app/views/question_views.py

```diff
@@ -143,9 +144,19 @@ def create_question(request: HttpRequest) -> HttpResponse:
             Lang2ID = get_languages() 
             language_id = Lang2ID.get(language, None)
 
+
             if language_id is None:
                 return JsonResponse({'error': 'Invalid language'}, status=400)
-        
+            
+            if (random() < 0.5):
```

Comment:
I deleted this line in my commit. If there is a reasonable answer to this, I am all ears.